analyses/molecular-subtyping-MB/05-subtype-mb-shh.R

```diff
@@ -0,0 +1,389 @@
+# Author: Ryan Corbett & Jo Lynne Rokita
+# Function: Script to subtype MB SHH tumors 
+
+# Load packages
+suppressPackageStartupMessages({
+  library(tidyverse)
+  library(data.table)
+})
+
+# Set directory paths
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set file paths
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "molecular-subtyping-MB")
+results_dir <- file.path(analysis_dir, "results")
+input_dir <- file.path(analysis_dir, "input")
+plots_dir <- file.path(analysis_dir, "plots")
+
+# set file paths
+hist_file <- file.path(data_dir, "histologies-base.tsv")
+mb_file <- file.path(results_dir, "MB_molecular_subtype.tsv")
+cn_file <- file.path(data_dir, "consensus_wgs_plus_cnvkit_wxs_plus_freec_tumor_only_autosomes.tsv.gz")
+maf_file <- file.path(data_dir, "snv-consensus-plus-hotspots.maf.tsv.gz")
+tumorOnly_file <- file.path(data_dir, "snv-mutect2-tumor-only-plus-hotspots.maf.tsv.gz")
+expr_file <- file.path(data_dir, "gene-expression-rsem-tpm-collapsed.rds")
+cn_arm_file <- file.path(data_dir, "cnv-consensus-gistic.zip")
+germline_file <- file.path(input_dir, "pbta-merged-plp-variants-cbtn-plus-mb-x01.tsv")
```

Comment:
does this include low frequency PLP variants? were any of note for this subtyping?

Reply:
I just checked the full results and all PTCH1, SUFU, ELP1, and TP53 P/LP variants had VAFs > 0.2